plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
I did mean to put that in a separate PR. The main issue is, if you try run a YAML/JSON with, say, 5 images then on my wifi the DCs fail to deploy because the images don't download in time. Deployments don't have this issue - there's no timeout for downloading the images - like there is for DCs. Having DCs that don't deploy is kinda a massive UX fail really.

So to make D and DCs work the same I figure we should give folks more than 10 minutes to download all possible images relating to a YAML / JSON and start them. 

I guess if you're not gonna buy my arguments about making DCs behave more like Ds OOTB; we can always just add this property to every pom.xml in every fabric8 repository ;) - but having DCs that typically don't work at all on slow-ish networks sounds kinda wrong to me. I guess a better long term fix is to get openshift not to fail a DC if its still downloading the image